source/_integrations/amcrest.markdown

```diff
@@ -20,20 +22,18 @@ There is currently support for the following device types within Home Assistant:
 - Binary Sensor
 - Camera
 - Sensor
+- Events
 
 ## Configuration
 
-To enable your camera in your installation, add the following to your `configuration.yaml` file:
+For configuration go to the `Integrations pane` on your Home Assistant instance.
 
-```yaml
-# Example configuration.yaml entry
-amcrest:
-  - host: IP_ADDRESS_CAMERA
-    username: YOUR_USERNAME
-    password: YOUR_PASSWORD
-
-```
+<div class='note'>
+  The  amcrest integration reads the previous configurations into the configuration.yaml file.
+It is recommended that you remove these configurations after import.
+</div>
 
```

Comment:
This is part of the breaking changes sections in the release notes, not or documentation.

Please remove this part.

```suggestion
```